third-party/massivethreads/Makefile

```diff
@@ -5,6 +5,19 @@ endif
 CHPL_MAKE_HOST_TARGET = --target
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
+#
+# Cray X* builds are cross-compilations.
+#
+ifneq (, $(filter cray-x%,$(CHPL_MAKE_TARGET_PLATFORM)))
+CHPL_MASSIVETHREADS_CFG_OPTIONS += --host=x86_64-cle-linux-gnu
+endif
+
+CHPL_MASSIVETHREADS_CFG_OPTIONS += --enable-malloc-wrapper=no \
+                                   --enable-eco-mode=no \
+                                   --enable-bind-workers=no
+
+CHPL_MASSIVETHREADS_CFG_OPTIONS += $(CHPL_MASSIVETHREADS_MORE_CFG_OPTIONS)
```

Comment:
Yes, this is to make it easy for a developer (typically an internal one) to pass additional options to the configure script, without editing the Chapel Makefile for that third-party package. So this would be set in the environment or on the make command line. Several of the packages already had this, and I'm adding to these that didn't have it before, while I'm here.